setup.cfg

```diff
@@ -29,18 +29,21 @@ include_package_data = True
 packages = find:
 install_requires =
     django
+    djangorestframework
+    drf-spectacular
```

Comment:
Can we drop `drf-spectacular` from the hard dependencies?

In the `apps.py` you can do a check before importing the extensions: `if apps.is_installed("drf_spectacular"): ...`